src/encoded/static/components/navigation/NavigationBar.js

```diff
@@ -70,7 +79,24 @@ export class NavigationBar extends React.PureComponent {
      * @returns {void}
      */
     componentDidMount(){
-        this.setState({ 'mounted' : true });
+        const { href } = this.props;
+
+        let testWarning = true;
+        let lsResult = null;
+
+        if (NavigationBar.inferCookieAcceptanceFromLocalStorage(href)){
+            if (window && window.localStorage && window.localStorage.getItem){
+                lsResult = window.localStorage.getItem("accepted_privacy_policy_disclaimer");
+                if (lsResult) {
+                    lsResult = new Date(lsResult);
```

Comment:
Needs cross-browser testing since apparently "Date" parsing is dependent on browser/JS-engine implementation. Might switch to using moment for this instead (supposed to.. not be implementation dependent).

Reply:
See note under `Date.parse` (implicitly called by `new Date(param)`) [at MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#Methods).

Reply:
Purpose of storing datetime instead of "true" is to be able to nullify acceptance + inform in future if policy changes.